src/nav/components.yml

```diff
@@ -80,9 +80,9 @@
       id: https://zendeskgarden.github.io/react-components/modals
     - title: Notification
       items:
-        - id: https://zendeskgarden.github.io/react-components/notifications
+        - id: /components/alert
```

Comment:
Likewise, there are multiple types of Alerts, should this be plural?

Reply:
I agree. This should be `Alerts`